project-files/startup/latest.xml

```diff
@@ -0,0 +1,4 @@
+<tdsVersions>
+    <version name="maintenance" value="4.6.21"/>
```

Comment:
haha this isn't right though. Although I don't know if we even use this value...

Reply:
yeah, definitely! I just updated to current development/release versions.

The `latest.xml` is retrieved from nexus in the `TdsUpdateConfigBean.getLatestVersionInfo` and then written  to the catalogInit.log file.

Reply:
Not sure why this is needed, just one more place to keep the versions up to date... so I would also be okay with removing it.

Reply:
I updated the latest.xml on the artifacts server as well as a test for retrieving that